cpp/tests/spatial/point_in_polygon_test.cpp

```diff
@@ -222,6 +223,7 @@ TYPED_TEST(PointInPolygonTest, 31PolygonSupport)
                 return  1.0;
             case 3:
             case 4:
+            default:
```

Comment:
Should probably be `CUSPATIAL_FAIL`, but it makes absolutely no difference. TBH I should have written it as
```
case 1:
case 2:
  return  1.0;
default:
  return -1.0;
```

Reply:
I considered that too, but left the current behavior in place since the behavior is more explicit.